woocommerce/woocommerce-ios
# Open orders when tapping orders part from home screen widget

## Description


When tapping Woo home screen widget the app gets opened. There was an option added in WidgetKit where we can link to a specific part of the app by either using `widgetURL` SwiftUI modifier or `Link` - https://developer.apple.com/documentation/widgetkit/linking-to-specific-app-scenes-from-your-widget-or-live-activity.
Since we are showing orders section in the widget I added a `Link` for that part of the widget and when users tap on that part of the widget the app will open and switch to the Orders.

pe5pgL-5zJ-p2

## Testing information


- Add Woo home screen widget
- Tap any other part of the widget (but not the Orders part) and check that the app is opened
- Tap Orders part of the widget and check that the orders tab is opened in the app

## Screenshots


https://github.com/user-attachments/assets/8c5a1f32-6e22-4b32-8459-34a8d064646d

![Simulator Screen Recording - iPhone 15 Pro - 2024-08-23 at 12 29 03](https://github.com/user-attachments/assets/54d27d9b-9d19-4798-a1fe-226c6e35a8ce)

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
